tests/Rules/Operators/OperandInArithmeticPostDecrementRuleTest.php

```diff
@@ -22,10 +22,6 @@ protected function getExpectedErrors(): array
 				'Only numeric types are allowed in post-decrement, false given.',
 				21,
 			],
-			[
-				'Only numeric types are allowed in post-decrement, string given.',
```

Comment:
We need to keep this error. It doesn't do anything: https://3v4l.org/3b3hf

Reply:
Indeed, I changed.